apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -0,0 +1,263 @@
+import { useCallback, useEffect, useState } from 'react';
+import {
+  Badge,
+  Box,
+  Flex,
+  TextLink,
+  Note,
+  Card,
+  Stack,
+  Paragraph,
+  FormControl,
+  Spinner,
+  Button,
+} from '@contentful/f36-components';
+import { ExternalLinkTrimmedIcon, CheckCircleIcon, ErrorCircleIcon } from '@contentful/f36-icons';
+import { useApi } from 'hooks/useApi';
+import { ServiceAccountKeyId, ServiceAccountKey } from 'types';
+import { ApiError } from 'services/api';
+
+interface Props {
+  serviceAccountKeyId: ServiceAccountKeyId;
+  serviceAccountKey: ServiceAccountKey;
+  onEditGoogleAccountDetails: React.MouseEventHandler<HTMLButtonElement>;
+  onAccountSummariesChange: Function;
+}
+
+export const GoogleErrorApiErrorTypes = {
+  AdminAPI: 'AdminApiError',
+  DataAPI: 'DataApiError',
+  Unknown: 'Unknown',
+};
+
+interface GAErrorApiType {
+  details: any;
+  errorType: string;
+  message: string;
+}
+
+const DisplayServiceAccountCard = (props: Props) => {
+  const {
+    serviceAccountKeyId,
+    serviceAccountKey,
+    onEditGoogleAccountDetails,
+    onAccountSummariesChange,
+  } = props;
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [adminApiError, setAdminApiError] = useState<GAErrorApiType>();
+  const [dataApiError, setDataApiError] = useState<GAErrorApiType>();
+  const [unspecifiedAdminError, setUnspecifiedAdminError] = useState<GAErrorApiType>();
+  const [unspecifiedDataError, setUnspecifiedDataError] = useState<GAErrorApiType>();
+
+  // NOTE: Due to a bug installation parameters are not available at sdk.parameters.installation form the config screen
+  // location. Therefore we must pass down the values directly to the useApi hook. If the bug is fixed this won't be
+  // necessary
+  const api = useApi(serviceAccountKeyId, serviceAccountKey);
+
+  const isApiGA4Error = (error: any): error is ApiError => {
+    if (!('errorType' in error && typeof error.errorType === 'string')) return false;
```

Comment:
I like to leave braces out for one liner if clauses - especially when there's multiple checks like on line 59 and 60. Let me know if this bothers people and I'll put them back in